README.md

```diff
@@ -7,20 +7,70 @@
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/simple-icons-font/font/simple-icons.min.css"
+          type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
+```
 
-### General Usage
+### Manual download
+
+1. Download the latest version of the package from
+[here][npm-registry-tarball-link], extract it and move the content of the
+`font` folder to your desired location. 
+1. Reference the CSS file in a `link` tag of your HTML:
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="simple-icons.min.css" type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
+```
 
-This package can be installed into your project with NPM to provide all of our icons as an icon font.
+### Using module bundlers
 
-```bash
+```
 npm install simple-icons-font
 ```
 
-After installation, the icon font can then be found in `node_modules/simple-icons-font/font` along with a stylesheet for
-a basic setup.
+After installation, the icon font can then be found in
+`node_modules/simple-icons-font/font` along with stylesheets for a basic setup.
+
+## Usage
+
+The CSS files of simple-icons-font contains next classes:
+
+- `simpleicons`: Adds the font to the container that will display the icon.
+- `simpleicons--color`: Colorizes the icon with the main color of the brand.
+- `simpleicons-<ICON SLUG>`: Replace here `<ICON SLUG>` by with the
+ correspondent slug for the icon. Their slugs correpond to filenames, which you
+ can see listed [here][simple-icons--icons-dir-link]. 
 
 ## Status
 
-[![Build Status](https://github.com/simple-icons/simple-icons-font/workflows/Verify/badge.svg)](https://github.com/simple-icons/simple-icons-font/actions?query=workflow%3AVerify+branch%3Adevelop)
-[![npm version](https://img.shields.io/npm/v/simple-icons-font.svg)](https://www.npmjs.com/package/simple-icons-font)
+[![Build Status][build-status-image]][build-status-link]
+[![NPM version][npm-version-image]][npm-package-link]
+
+[build-status-image]: https://img.shields.io/github/workflow/status/simple-icons/simple-icons-font/Verify/develop?logo=github
+[build-status-link]: https://github.com/simple-icons/simple-icons-font/actions?query=workflow%3AVerify+branch%3Adevelop
+[npm-version-image]: https://img.shields.io/npm/v/simple-icons-font?logo=npm
+[npm-package-link]: https://www.npmjs.com/package/simple-icons-font
+[simple-icons--icons-dir-link]: https://github.com/simple-icons/simple-icons/tree/develop/icons
+[npm-registry-tarball-link]: https://registry.npmjs.org/simple-icons-font/-/simple-icons-font-2.0.0.tgz
```

Comment:
I would really be against having to update this for *every* release rather than every *major* release. Can we link to a page with all tarballs instead?

Reply:
Do you know how to get the directory where NPM serves all tarballs? :file_folder:
I have not found it anywhere...

Reply:
Hmm :thinking: We could add [automated GitHub releases](https://github.com/simple-icons/simple-icons/blob/18cac9da6f091444bc391467067183ab2336b0af/.github/workflows/publish.yml#L28-L73) and link to the latest instead? E.g.

```
https://github.com/simple-icons/simple-icons/releases/latest
```

Currently links to:


```
https://github.com/simple-icons/simple-icons/releases/tag/4.2.0
```


And is automatically updated each release.

Reply:
Yes, would be the solution. I keep it like this and update when we add automated releases?

Reply:
I will open a PR to set up automated releases later, we can see in which PR we change it later :+1:

Reply:
This will be unresolved until #54 merges. I think that we can wait for.

Reply:
Sorry, didn't notice this comment until just after I merged this PR :sweat_smile: I will make sure to include this change in #54